src/main/java/org/zalando/problem/spring/web/advice/AdviceTrait.java

```diff
@@ -87,13 +88,26 @@
         return create(status, throwable, request, new HttpHeaders());
     }
 
+    default ResponseEntity<Problem> create (final HttpStatus status, final Throwable throwable,
+            final NativeWebRequest request) throws HttpMediaTypeNotAcceptableException {
+        return create(status, throwable, request, new HttpHeaders());
+    }
+
     default ResponseEntity<Problem> create(final StatusType status, final Throwable throwable,
             final NativeWebRequest request, final HttpHeaders headers)
             throws HttpMediaTypeNotAcceptableException {
 
         final ThrowableProblem problem = toProblem(throwable, status);
         return entity(problem, request, headers);
     }
+
+    default ResponseEntity<Problem> create(final HttpStatus status, final Throwable throwable,
```

Comment:
Same here, I'd move this one method up and delegate to the other overloaded version.